x/ccv/consumer/keeper/genesis.go

```diff
@@ -17,6 +17,17 @@ import (
 //  2. A consumer chain restarts after a client to the provider was created, but the CCV channel handshake is still in progress
 //  3. A consumer chain restarts after the CCV channel handshake was completed.
 func (k Keeper) InitGenesis(ctx sdk.Context, state *consumertypes.GenesisState) []abci.ValidatorUpdate {
+
+	// PreCCV is true during the process of a sovereign to consumer changeover.
+	// At the PreCCV point in the process, the sovereign chain has just been upgraded to include
+	// the consumer ccv module, but the sovereign staking keeper is still managing the validator set.
+	// In two blocks, the consumer ccv module will take over proof of stake capabilities,
```

Comment:
Remove "In two blocks" as this may change once we add ABCI++. Add 
> Once the provider validator set starts validating blocks, the consumer CCV module ...

Reply:
See https://github.com/cosmos/interchain-security/commit/4f2e77e15089a20bf4967ae0710a465a6656eb9e